src/input_common/gcadapter/gc_poller.cpp

```diff
@@ -73,7 +79,7 @@ std::unique_ptr<Input::ButtonDevice> GCButtonFactory::Create(const Common::Param
     const auto button_id = params.Get("button", 0);
     const auto port = static_cast<u32>(params.Get("port", 0));
 
-    constexpr int PAD_STICK_ID = static_cast<u16>(GCAdapter::PadButton::PAD_STICK);
+    constexpr int PAD_STICK_ID = static_cast<u16>(GCAdapter::PadButton::Stick);
```

Comment:
```suggestion
    constexpr s32 PAD_STICK_ID = static_cast<s32>(GCAdapter::PadButton::Stick);
```

Any reason you're casting to u16 but storing it into an s32? Are you intending to clamp the value? If so, you should use std::clamp to avoid overflows and underflows